guides/Retrospectives.md

```diff
@@ -0,0 +1,71 @@
+
+
+1. Everyone gather in a room or zoom call (or both). _*_
+2. Set a timer for 10 minutes to write down items to go in the Glad, Sad, or Mad columns.
+3. Once the timer is finished, group similar items or topics if applicable.
+4. Vote on items that you think should be discussed. _* *_
+5. Go through and discuss each item starting with the item with the most votes.
+6. Generate and assign action items that come out of the discussion.
+7. (Optional) Discuss remaining items that may not have received votes if there is enough time
+
+
+_*_  Make sure to review the action items from the previous retrospective to determine if they were useful. Ideally this should come up as topic items but it may be helpful to specifically call them out when you first start conducting retros.
+
+_* *_ If this is in person typically everyone gets 3-5 votes (represented by stickers) to assign to topics they wish to discuss.
+
+
+__Action Items__
+Be sure to come up with action items that are actionable and can be performed by a team member or the whole team. An example of that might be:
+
+- **Issue or topic discussed**: Sentry React Native, which is supposed to collect crashes, was causing a crash. Entire team has felt this pain point and the team decided to search for an alternative.
+- **Action Item**: Yuki has agreed to do a spike to search for an alternative. This might be a task pulled in next sprint.
+
+
+__Tooling__
+At Artsy it is typically the case that team members are distributed across different countries/time zones. In order to conduct a retrospective that accommodates everyone, it may be necessary to use web-based tools such as zoom.us and [retrium.com](https://www.retrium.com/).
+
+
+## Things to keep in mind
+- It’s really important to cultivate a safespace for retros. Retros are most successful when all team members are able to voice their thoughts, concerns, and ideas.
+- In some cases the team may want to run a retro with a specific focus in mind. For example, the team could facilitate a retro for improving how the quality assurance process is done or managing technical debt. Another good opportunity for a retro is after the team has implemented a new stack or technology.
+- These are by no means hard rules. Rather, this is meant to serve as a guideline and give some initial structure that teams can then adjust to fit their needs.
+
+
+**Resources:**
+- [10 things all agile teams need to know to run effective retrospectives](https://blog.retrium.com/10-things-all-agile-teams-need-to-know-to-run-effective-retrospectives)
+- [Retrospective](https://www.atlassian.com/team-playbook/plays/retrospective)
+- [The Collector Experience & GMV team’s first retro](https://github.com/artsy/collector-experience/wiki/Our-First-Retro)
+
```

Comment:
I wrote a [blog post on iOS Retrospectives](http://artsy.github.io/blog/2017/05/27/ios-retrospectives/) that might be good to link to.

Reply:
I think that is a good example of a retro with a specific focus discussed above. We'll add a link to it.